tests/nemo_text_processing/en/data_text_normalization/test_cases_address.txt

```diff
@@ -7,3 +7,5 @@
 12 S 1st st~twelve South first Street
 1990 for the Ata ST~nineteen ninety for the Ata ST
 Main St.~Main St.
+Nancy lived at 1428 Elm St. It was a strange place.~Nancy lived at fourteen twenty eight Elm Street. It was a strange place.
+(incorrect normalization) Nancy lived at 1428 Elm St. 4 doors down lived her best friend.~(incorrect normalization) Nancy lived at fourteen twenty eight Elm Street four doors down lived her best friend.
```

Comment:
Does the tester allow this formatting?

Reply:
The tests (local and CI) pass without issues.

[Something similar](https://github.com/NVIDIA/NeMo-text-processing/blob/main/tests/nemo_text_processing/es/data_inverse_text_normalization/test_cases_ordinal.txt) was implemented for Spanish ITN.  @mgrafu suggested including that in lieu of documenting the edge case / limitation of the model.

Reply:
Oh no, in the future you want to document edge cases. No one is going to check data files beyond us.